### DISCUSSION TITLE
Add blocks grouping feature

### DISCUSSION THREAD

**Discussion by revnelson** [payloadcms/payload#5609] [Feature Requests & Ideas]
It'd be great to be able to group Blocks so we can, for example, have a UI blocks group and a Containers blocks group. Currently we've got UI blocks such as Buttons, Heros, WYSIWYGs (richText), etc. and then Container blocks such as Grid, Flex, Carousel, etc. It would make our editor's lives much easier to see groups so they can pick the appropriate block at a glance. A group field could be added to the admin config for blocks similar to collections.

**Comment by Sandro48** [comment]
+1 for this, if you have a lot of layout blocks, it just looks messy

**Comment by akhrarovsaid** [comment]
Yep, I agree. I've authored a [PR which addresses this here](https://github.com/payloadcms/payload/pull/11239).

**Comment by paulpopus** [comment]
PR has been merged, it will be available in the next release (3.25.0). Thanks everyone!
